string data
bool antenna_stop
bool dome_stop
bool dome_close
bool memb_close
string from_node
float64 timestamp